.github/workflows/close-org-prs.yml

```diff
@@ -0,0 +1,44 @@
+name: Close PRs from Orgs
+
+on:
+  pull_request_target:
+    types: [opened, reopened, synchronize]
+
+permissions:
+  pull-requests: write
+  issues: write
+
+jobs:
+  close-org-prs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if PR is from an org and close if so
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const pr = context.payload.pull_request;
+            const isFork = pr.head.repo.full_name !== pr.base.repo.full_name;
+            const forkOwnerType = pr.head.repo.owner.type;
+            if (isFork && forkOwnerType === 'Organization') {
+              // Add 'invalid' label
+              await github.rest.issues.addLabels({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                issue_number: pr.number,
+                labels: ['invalid']
+              });
+              // Close the PR
+              await github.rest.pulls.update({
+                owner: context.repo.owner,
```

Comment:
Are the `owner` and `repo` options needed for these calls? Seems like that's something the library might detect from the environment it's running in. Same with `issue_number` and `pull_number`.

Reply:
it's what they had in the example for github scripts action https://github.com/actions/github-script?tab=readme-ov-file#comment-on-an-issue but i'm happy to test it

Reply:
for the `pull_request_target` it didn't work without it for some reason I got 403 errors.